modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,53 @@
+= Share an Asset
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+:page-aliases: to-share-an-asset-with-a-user.adoc
+
+Exchange administrators can share an asset with another user or with a team in a separate organization.
```

Comment:
Exchange administrators can share an asset with another user within the same organization, or with the whole organization. They can also share an asset with an external organization.

>> We are not dealing with Teams in this release

Reply:
Thank you! Fixed.